src/loaders/hooks.js

```diff
@@ -5,9 +5,10 @@ export default ({ hooks }) => {
   const hookGroups = new DataLoader(queries =>
     Promise.all(
       queries.map(async ({ filter }) => {
-        const { groups } = await hooks.listHookGroups();
+        const { groups: raw } = await hooks.listHookGroups();
+        const hookGroups = filter ? sift(filter, raw) : raw;
```

Comment:
The sifter function will be provided to map against the return value of the method, so it is going to be expecting to have the `{ hookGroupId }` as its value. You'll need to map first:

```js
const { groups } = await hooks.listHookGroups();
const raw = groups.map(hookGroupId => ({ hookGroupId }));

return filter ? sift(filter, raw) : raw;
```